usaon_benefit_tool/templates/home.html

```diff
@@ -6,8 +6,8 @@
   <h2>{% block title %}Welcome to the US AON Benefit Tool!{% endblock %}</h2>
     <p><strong>Actions:</strong></p>
     <ul>
-      <li>View our <a href="https://usaon.org/apps/benefit-tool/surveys">library of evaluations here.</a></li>
-      <li>Click here to <a href="https://usaon.org/apps/benefit-tool/surveys">edit or update your own evaluation.</a></li>
+      <li>View our <a href="{{ url_for("surveys.view_surveys") }}">library of evaluations here.</a></li>
+      <li>Click here to <a href="{{ url_for("surveys.view_surveys") }}">edit or update your own evaluation.</a></li>
```

Comment:
This way the app can be agnostic about what URL it's deployed at :)

Reply:
Good catch!

Reply:
Wow, you're fast! :zap: